src/js/components/modal.js

```diff
@@ -114,7 +120,7 @@ class Modal {
   }
 
   onModalClick (event) {
-    if (this.$modal.is(event.target)) {
+    if (!this.options.static && this.$modal.is(event.target)) {
```

Comment:
This expression is so hard to read. Can you abstract this as a method? Like `isStatic` or something like?